src/Build/BackEnd/Components/RequestBuilder/TaskHost.cs

```diff
@@ -670,6 +670,25 @@ public void LogTelemetry(string eventName, IDictionary<string, string> propertie
 
         #endregion
 
+        #region IBuildEngine7 Members
+        /// <summary>
+        /// Enables or disables emitting MSB4134 to log when a task fails without logging errors
+        /// </summary>
+        public void SetEnableMSB4134(bool enableError)
```

Comment:
Why is this an explicit set method rather than a setter?

Reply:
You mean a property with `{get; set;}`? I don't think you can access that with reflection.

Reply:
You can, with [`Type.GetProperty`](https://docs.microsoft.com/dotnet/api/system.type.getproperty?view=netframework-4.8#System_Type_GetProperty_System_String_)